src/components/decision/classification/classification.ts

```diff
@@ -205,7 +206,8 @@ export class ClassificationComponent extends DecisionComponent {
       >
         <span class="oe-pill decision-color-pill" style="background-color: var(${color})"></span>
 
-        <div class="button-text">${decision}</div>
+        <div class="button-text">${toTitleCase(decision)}</div>
```

Comment:
are we sure we want to modify this text? Isn't that a concern of the supplier of the decision element?

Reply:
This is the classification component's "decision enum" value e.g. `true`, `false`, `unsure`, `skip`.

If you want to provide a "tag name" (to be shown above the classification buttons), you can still provide an element in the default slot.

e.g.

```html
<oe-classification tag="dog">
	Cat (Yes I know it's different)
</oe-classification>
```

In my example, none of the user-supplied content will run through this `toTitleCase` helper.

Reply:
fair enough